src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#" method="post">
+      <div class="search-bar search-bar__big" data-qa="big">
+        <label for="big">
+          <img
+            src="./images/Search.svg"
+            alt="search-logo"
+            class="search-bar__logo search-bar__logo--big"
```

Comment:
still not resolved 
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/55200530/de5c7e95-2d9f-46f7-95ba-8b8a1ad10801)